### ISSUE TITLE
BUG - Random seed in _plotting

### ISSUE THREAD

**Issue by andreacortis** [MaartenGr/BERTopic#952]
https://github.com/MaartenGr/BERTopic/blob/06dcd47a019854185a39146ed693d16dc3a27651/bertopic/plotting/_topics.py#L65

```
embeddings = UMAP(n_neighbors=2, n_components=2, metric='hellinger').fit_transform(embeddings)
```

Right now every time you visualize the intertopic distance you get a new visualization. This is because the UMAP instance does not have a fixed random seed.

**Comment by MaartenGr** [comment]
Thanks for sharing this. The difficulty here is that it might hurt the performance of UMAP when a `random_state` is set which is why I initially decided not to use that here. I might change it but I will have to test the quality of the output as well as the wall time for larger datasets.
